hirte.spec.in

```diff
@@ -0,0 +1,57 @@
+Name:		hirte
+Version:	@VERSION@
+Release:	@RELEASE@%{?dist}
+Summary:	Container orchestrator for edge based clusters
+License:	GPLv2
+URL:		https://github.com/containers/hirte
+Source0:	%{name}-%{version}.tar.gz
+
+BuildArch:	x86_64 aarch64
+
+BuildRequires:	gcc
+BuildRequires:	meson
+BuildRequires:	systemd-devel
+
+Requires:	systemd
+
+
+%description
+This package contains the Hirte container orchestrator for edge based clusters.
+
+
+%package agent
+Summary:	Hirte container orchestrator agent
+Requires:	systemd
+
+
+%description agent
+This packages contains the agent for Hirte container orchestrator.
```

Comment:
"This package contains the agent for Hirte."

Reply:
Just now aligned with comments from https://github.com/containers/hirte/pull/97